chain/network/src/stable_hasher.rs

```diff
@@ -0,0 +1,18 @@
+/// Todo consider moving this into a separate crate.
+
+#[allow(deprecated)]
+use std::hash::{Hasher, SipHasher};
+
+/// We not use stable hasher as it could change with Rust releases, so rely on stable SIP hash.
+#[allow(deprecated)]
+#[derive(Default, Clone)]
+pub(crate) struct StableHasher(SipHasher);
+
+impl Hasher for StableHasher {
+    fn finish(&self) -> u64 {
+        self.0.finish()
+    }
+    fn write(&mut self, bytes: &[u8]) {
+        self.0.write(bytes)
+    }
+}
```

Comment:
I am a bit confused by this. What is the reason for not using `Hasher` directly?

Reply:
usage of `SipHasher` directly is deprecated. I think we use wrapped to avoid using `#[allow[depraceted]]` in another places.
see `runtime/near-vm-runner/src/cache.rs`

Reply:
Should I create a separate issue for moving `StableHasher` to separate package.

Reply:
I moved it to `near-stable-hasher` crate.